process/track/sovereignChainBlockProcessor.go

```diff
@@ -117,3 +118,11 @@ func (scbp *sovereignChainBlockProcessor) requestHeadersIfNothingNewIsReceived(
 
 	scbp.baseRequestHeadersIfNothingNewIsReceived(lastNotarizedHeaderNonce, latestValidHeader, highestRoundInReceivedHeaders, shardID)
 }
+
+func (scbp *sovereignChainBlockProcessor) getBlockFinality(shardID uint32) uint64 {
+	if shardID == core.MainChainShardId {
+		return 0
```

Comment:
block finality isn't 1 for mainchain now ?

Reply:
Yes, it is, but notifier only sends finalized headers, w.r.t to this comment:

> Whenever receving a new incoming header, the resulted header will be instantly included in the sovereign block. Reason being that notifier only sends finalized mainchain headers, so we do not need block finality = 1 when notarizing incoming headers